src/storages/AbstractSplitsCacheAsync.ts

```diff
@@ -17,6 +18,7 @@ export abstract class AbstractSplitsCacheAsync implements ISplitsCacheAsync {
   abstract getChangeNumber(): Promise<number>
   abstract getAll(): Promise<ISplit[]>
   abstract getSplitNames(): Promise<string[]>
+  abstract getNamesByFlagSets(flagsets: string[]): Promise<ISet<string>>
```

Comment:
Emma, replace also the occurrences of `flagsets` to `flagSets`. Include others like `flagsetsFilter` to `flagSetsFilter`, etc.